src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + getArea() + " sq. units, "
+                + "radius: " + radius + "units, "
+                + "color:" + getColor());
```

Comment:
You are using a string concatenation in a print statement. This is a performance issue. It would be better to use a StringBuilder to construct the string, and then print it.